assets/app/views/_volumes.html

```diff
@@ -8,13 +8,19 @@
   <dl class="dl-horizontal left">
     <div ng-if="volume.secret">
       <dt>Type:</dt>
-      <dd>secret</dd>
+      <dd>
+        secret
+        <span class="small text-muted">(a volume populated by a Secret)</span>
```

Comment:
feels redundant to repeat the word volume here, "populated by a Secret at container creation" maybe?